cpp/src/arrow/ipc/json_integration_test.cc

```diff
@@ -54,11 +54,6 @@ using internal::TemporaryDir;
 
```

Comment:
Perhaps you should add some metadata in the JSON examples of this file?

Reply:
done